insights/client/connection.py

```diff
@@ -685,7 +700,7 @@ def _fetch_system_by_machine_id(self):
                 url = self.base_url + '/platform/inventory/v1/hosts?insights_id=' + machine_id
             else:
                 url = self.inventory_url + '/hosts?insights_id=' + machine_id
-            res = self.get(url)
+            res = self.get(url, log_response_text=False)
```

Comment:
hm this is one of the main queries the client does to verify its own identity; I'd definitely want to keep this logged, to better diagnose possible issues with HBI

Reply:
I have my doubts about the usefulness of logging this, but I can see the point.